.github/workflows/dependencies.yml

```
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Check for unused dependencies
+on: [push]
```

```suggestion
on:
  push:
    paths: 'offchain/**'
```
As we did with the code format workflow, shouldn't we run this only if there were changes in the offchain directory? It might be even more specific and only trigger if there were changes in Cargo.toml files 🤔